conda_smithy/configure_feedstock.py

```diff
@@ -319,6 +322,7 @@ def main(forge_file_directory):
     recipe_dir = 'recipe'
     config = {'docker': {'image': 'condaforge/linux-anvil', 'command': 'bash'},
               'templates': {'run_docker_build': 'run_docker_build_matrix.tmpl'},
+              'ci_enabled': [],
```

Comment:
Wouldn't that make it a user tunable parameter in two places though? Or should we remove it before `conda-forge.yml` is written?